packages/thirdweb/src/react/web/ui/ConnectWallet/screens/Buy/fiat/FiatSteps.tsx

```diff
@@ -102,8 +114,100 @@ export function FiatSteps(props: {
     fromCurrencySymbol,
     fromCurrencyAmount,
     toTokenAmount,
+    intentId,
   } = props.partialQuote;
 
+  const account = useActiveAccount();
+  const switchChain = useSwitchActiveWalletChain();
+
+  const deploySmartWallet = async ({ chain }: { chain: Chain }) => {
+    if (!account) return;
+
+    await switchChain(chain);
+
+    const smartWalletHandle = smartWallet({
+      chain,
+      sponsorGas: true,
+    });
+
+    const smartAccount = await smartWalletHandle.connect({
```

Comment:
Right now, this creates a smart wallet that is deterministic to the account, but **not** per intent. How can I send in an intentId as the string to be deterministic by?